### ISSUE TITLE
Not able to use Horizon with phpredis out of the box

### ISSUE THREAD

**Issue by bartabel** [laravel/horizon#455]
I am not able to use Horizon with phpredis on Centos. I will get the following errors:
"Call to undefined method Illuminate\Support\Facades\Redis::connect()"
(I have no issues when using predis instead of phpredis).

I know that there are various old issues for framework that have been closed stating that this is not a bug and that the Redis Facade alias should be changed. Changing this indeed solves the issue, but what is the side effect? Can the Redis alias be changed without consequences?

So, in short, if it supposed to work with PhpRedis, then it is not really clear from the documentation how to set it up (with standard installs of Redis and Phpredis through yum). So either see this as a bug report on the documentation or the code.

**Comment by bartabel** [comment]
Got it working at last with the changed Alias, but I still like to know if this is indeed the preferred solution.

**Comment by driesvints** [comment]
Hi there,

Looks like this is a question which can be asked on a support channel. Please only use this issue tracker for reporting bugs with the library. If you have a question on how to use functionality provided by this repo you can try one of the following channels:

- [Laracasts Forums](https://laracasts.com/discuss)
- [Laravel.io Forums](https://laravel.io/forum)
- [Discord](https://discordapp.com/invite/KxwQuKb)
- [Larachat](https://larachat.co)
- [IRC](https://webchat.freenode.net/?nick=laravelnewbie&channels=%23laravel&prompt=1)
- [StackOverflow](https://stackoverflow.com)

**Comment by bartabel** [comment]
Why not add it to the documentation? I think it is a very common use case. Support channels are not very active on Horizon questions.

**Comment by bartabel** [comment]
> > Got it working at last with the changed Alias, but I still like to know if this is indeed the preferred solution.
> 
> > Why not add it to the documentation?
> 
> Can you elaborate what exactly your solution was?

The solution was to rename the default "Redis" alias that is created in config/app.php to "RedisConnection" (or anything other name).

**Comment by mfn** [comment]
Ah yes, makes sense. It shadows the actual global class. Technically there's no other solution, only a) don't use or rename the alias or b) education (docs update).

Thanks for clarifying!
